src/Nightingale.Converters/Nightingale.Converters.csproj

```diff
@@ -7,6 +7,7 @@
   <ItemGroup>
     <PackageReference Include="JeniusApps.Insomnia.NET" Version="0.0.3-preview" />
     <PackageReference Include="JeniusApps.Postman.NET" Version="0.2.4-preview" />
+    <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
```

Comment:
Out of curiosity what is this package for?

Reply:
Without this package I got https://docs.microsoft.com/en-us/dotnet/csharp/misc/cs0656 when compiling converter project due to `dynamic` in insomnia converter. I wonder if this is me only? If so I can remove it.

I should have mentioned CS0656 in the commit😅

Reply:
Oh yes you're right thanks for adding that!